libr/asm/p/asm_x86_nz.c

```diff
@@ -1887,22 +1892,23 @@ static int parseOperand(RAsm *a, const char *str, Operand *op) {
 		RFlagItem *flag;
 		op->reg = parseReg (a, str, &nextpos, &op->type);
 
-		if (op->reg == X86R_UNDEFINED && a->num) {
-			RCore *core = (RCore *)(a->num->userptr);
-			op->is_good_flag = true;
-			if (!(flag = r_flag_get (core->flags, str))) {
-				op->is_good_flag = false;
+		if (op->reg == X86R_UNDEFINED) {
+			op->is_good_flag = false;
+			if (!(a->num)) {
 				return nextpos;
 			}
 			op->type = OT_CONSTANT;
+			RCore *core = (RCore *)(a->num->userptr);
```

Comment:
userptr can be NULL. and in fact it can be anything, not just RCore..but we can asume this for now i guess... the problem is that this thing will crash if used without an rcore instance